assets/src/components/repos/installer/Installer.tsx

```diff
@@ -178,7 +178,7 @@ export function Installer({ setOpen, setConfirmClose, setVisible }) {
 
     setStepsLoading(true)
     build().then(() => setStepsLoading(false))
-  }, [client, selectedApplications.length])
+  }, [client, selectedApplications, selectedApplications.length])
```

Comment:
Since `selectedApplications` was added we no longer need `selectedApplications, selectedApplications.length` I think.